Error: prefix already exists: /home/ubuntu/miniconda/envs/_pipbuild_
Collecting oslo.serialization==1.6.0
  Could not find a version that satisfies the requirement oslo.serialization==1.6.0 (from versions: 0.1.0, 0.2.0, 0.3.0, 1.0.0, 1.1.0, 1.2.0, 1.3.0, 1.4.0, 1.5.0)
No matching distribution found for oslo.serialization==1.6.0
Package search: {'_pypi_ordering': False, 'version': '1.6.0', 'name': 'oslo.serialization', 'summary': 'Oslo Serialization library'}
Creating standard recipe for oslo.serialization-1.6.0
Using url https://pypi.python.org/packages/source/o/oslo.serialization/oslo.serialization-1.6.0.tar.gz (23 KB) for oslo.serialization.
Downloading oslo.serialization
Using cached download
Unpacking oslo.serialization...
done
working in /tmp/tmp_wo8muconda_skeleton_oslo.serialization-1.6.0.tar.gz
Fetching package metadata: ......

WARNING: conda-build appears to be out of date. You have version 1.13.0 but the
latest version is 1.14.0. Run

conda update -n root conda-build

to get the latest version.

Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).
Traceback (most recent call last):
  File "setup.py", line 18, in <module>
    import setuptools
ImportError: No module named setuptools

Applying patch: '/tmp/tmp_wo8muconda_skeleton_oslo.serialization-1.6.0.tar.gz/pypi-distutils.patch'
$PYTHONPATH = /tmp/tmp_wo8muconda_skeleton_oslo.serialization-1.6.0.tar.gz/oslo.serialization-1.6.0
Error: command failed: /home/ubuntu/miniconda/envs/_build/bin/python setup.py install

conda create -n _pipbuild_ --yes python pip
/home/ubuntu/miniconda/envs/_pipbuild_/bin/pip install oslo.serialization==1.6.0
Error: Could not pip install oslo.serialization==1.6.0
